Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM gradle:jdk17-alpine AS build
+# copy source to image
+WORKDIR /app
+COPY . /app
+ENV GRADLE_OPTS="-Dorg.gradle.daemon=false"
+# build jar by gradle
+RUN gradle build -x test
+
+
+# Run stage
+FROM openjdk:17-jdk-bullseye
+WORKDIR /app
+COPY --from=build /app/build/libs/be-0.0.1-SNAPSHOT.jar app.jar
```

Comment:
if app_name is foodbe, it should be "COPY --from=build /app/build/libs/foodbe-0.0.1-SNAPSHOT.jar app.jar"

Reply:
Try to build it locally before write to dockerfile